administrator/modules/mod_multilangstatus/src/Helper/MultilangstatusAdminHelper.php

```diff
@@ -95,11 +96,29 @@ public static function getState(CMSApplication $app, DatabaseInterface $db)
 	 */
 	public static function publish(CMSApplication $app, DatabaseInterface $db)
 	{
-		// If the module is trashed do not change its status
-		if (self::getState($app, $db) != -2)
+		// Get module
+		$module = ModuleHelper::getModule('mod_multilangstatus');
+
+		// Get module state.
+		if (empty($module->id))
+		{
+			$state = self::getState($app, $db);
+		}
+		else
+		{
+			$state = 1;
+		}
+
+		// If the module is trashed or missed do not change its status
+		if ($state == -2 || $state === null)
```

Comment:
please use type safe operators
```suggestion
		if ($state === -2 || $state === null)
```

Reply:
Not my code...